.circleci/config/executors/@executors.yml

```diff
@@ -1,10 +1,10 @@
 go-machine:
   machine:
-    image: 'ubuntu-2004:202107-02'
+    image: 'ubuntu-2004'
```

Comment:
Note: This is required, as the old image `ubuntu-2004:202107-02` was deprecated by circleci.